app/app.js

```diff
@@ -14,19 +14,20 @@
  * limitations under the License.
  */
 
-import Ember from 'ember';
-import Resolver from 'ember/resolver';
-import loadInitializers from 'ember/load-initializers';
-import config from './config/environment';
+// import Ember from 'ember';
+const Ember = require('ember');
```

Comment:
Can you explain these changes?

Reply:
Because of import/extension rule.

Reply:
Can you point to this rule? Looking at our [es6 style guide](https://github.com/Kinvey/style-guides/blob/master/backend/es6.md), it contains _many_ examples of using `import`, and I can't find a clause that says they shouldn't be used.

Reply:
https://github.com/Kinvey/eslint-kinvey-platform/blob/master/kinvey-rules.js#L781

Reply:
Well you are right, no objections. But I'm following the ruleset, not the style guide...I was told that this is the final ruleset, which seems to be not the truth. I saw that you already opened an discussion regarding these differences, which I hope to solve as soon as possible. Meanwhile, what we will do with that PR?

Reply:
I think either we remove these changes, or we leave this PR until we decide on the proper style guide. These changes are not critical to the component (_particularly_ this one even more so than others), and it does not make sense to do them just because somehow a rule got added, by mistake, to the lint config.

Reply:
Changes removed. import/extensions rule disabled until further clarification. Please, take another look.